src/Components/Publishing/ToolTip/ArtistToolTip.tsx

```diff
@@ -120,6 +122,10 @@ export const ArtistContainer = styled.div`
       color: black;
     }
```

Comment:
I'm wondering if it makes sense to make this `new feature, tell us what you think` part of market data component. What you now have in `<NewFeature />` component - does that refer to market data or the whole roll over in general?